Microsoft.Toolkit.Uwp.UI.Controls/ImageCropper/ImageCropper.cs

```diff
@@ -352,9 +354,53 @@ public async Task LoadImageFromFile(StorageFile imageFile)
         /// Gets the cropped image.
         /// </summary>
         /// <returns>The cropped image.</returns>
-        public async Task<WriteableBitmap> GetCroppedBitmapAsync()
+        public async Task<WriteableBitmap> GetCroppedImageAsync()
         {
-            return await GetCroppedBitmapAsync(Source, _currentCroppedRect);
+            if (Source == null)
+            {
+                return null;
+            }
+
+            var writeableBitmap = new WriteableBitmap((int)Math.Floor(_currentCroppedRect.Width), (int)Math.Floor(_currentCroppedRect.Height));
+            using (var randomAccessStream = new InMemoryRandomAccessStream())
+            {
+                var bitmapEncoder = await BitmapEncoder.CreateAsync(BitmapEncoder.PngEncoderId, randomAccessStream);
+                await CropImageAsync(Source, _currentCroppedRect, bitmapEncoder);
+                writeableBitmap.SetSource(randomAccessStream);
+            }
+
+            return writeableBitmap;
+        }
+
+        /// <summary>
+        /// Saves the cropped image to a file with the specified format.
+        /// </summary>
+        /// <param name="file">The target file.</param>
+        /// <param name="bitmapFileFormat">the specified format.</param>
+        /// <returns>Task</returns>
+        public async Task SaveAsync(StorageFile file, BitmapFileFormat bitmapFileFormat)
+        {
+            using (var fileStream = await file.OpenAsync(FileAccessMode.ReadWrite, StorageOpenOptions.None))
+            {
+                await SaveAsync(fileStream, bitmapFileFormat);
+            }
+        }
+
+        /// <summary>
+        /// Saves the cropped image to a stream with the specified format.
+        /// </summary>
+        /// <param name="stream">The target stream.</param>
+        /// <param name="bitmapFileFormat">the specified format.</param>
+        /// <returns>Task</returns>
+        public async Task SaveAsync(IRandomAccessStream stream, BitmapFileFormat bitmapFileFormat)
+        {
+            if (Source == null)
+            {
+                return;
+            }
+
+            var bitmapEncoder = await BitmapEncoder.CreateAsync(GetEncoderId(bitmapFileFormat), stream);
```

Comment:
Add `AsTask().ConfigureAwait(false);`

Reply:
@skendrot we probably need to srub our APIs, I don't think we do this everywhere, eh?

Reply:
We should be doing a fairly decent job of using `ConfigureAwait`

Reply:
We shouldn't do that. It won't work here.
Thread synchronization is required or an exception will throw.
>System.Exception:“The application called an interface that was marshalled for a different thread. (Exception from HRESULT: 0x8001010E (RPC_E_WRONG_THREAD))”

Reply:
This is wrong. We do need the context (i.e. UI) for our next operation so `ConfigureAwait(false)` shouldn't be used here.